config/manager/tcs_issuer.yaml

```diff
@@ -46,12 +52,15 @@ spec:
         - --user-pin=$USER_PIN
         - --so-pin=$SO_PIN
         - --use-random-nonce=true # Disable this if using KMRA version < 2.2
-        image: tcs-issuer
+        image: tcs-issuer:0.0.0
```

Comment:
Is version 0.0.0 right here?

Reply:
This is to satisfy the `trivy` tool. Otherwise, it does not matter as it gets replaced in the final deployment manifest either with the 'latest' or the release version.